src/Uno.Foundation/Metadata/ApiInformation.shared.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.Foundation.Metadata
+{
+	public partial class ApiInformation
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)